python-package/lightgbm/basic.py

```diff
@@ -638,6 +639,8 @@ def _lazy_init(self, data, label=None, max_bin=None, reference=None,
         self.predictor = predictor
         if self.max_bin is not None:
             params["max_bin"] = self.max_bin
+            warnings.warn('The `max_bin` parameter is deprecated and will be removed in next version. '
```

Comment:
I think `next` may not be clear. Can we give a specific version number?

Reply:
@wxchan Good idea! What about **2.0.12**?

Reply:
@StrikerRUS ok. but we might jump some version. @guolinke can confirm this.